### PR TITLE
Add WebdriverIO and Cucumber acceptance test framework

### PR DESCRIPTION
**Pull request by ScenarioFactory**

### CODE REVIEW

File: test/acceptance/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+node_modules
+reports
+test-output
+.dockerignore
```
Comment (swdpcomputing) [inline test/acceptance/.dockerignore:3]: Why is .dockerignore in the .dockerignore?

Comment (ScenarioFactory) [reply test/acceptance/.dockerignore:3]: That's a good question.. refactored file

**Review by swdpcomputing — DISMISSED** [review]

**Review by Andrew-Folga — DISMISSED** [review]
LGTM

File: test/acceptance/.dockerignore (hunk 1 above)
Comment (alanplatt) [reply test/acceptance/.dockerignore:3]: It should be in there, you don't need to copy the .dockerignore file into the container

Comment (ScenarioFactory) [reply test/acceptance/.dockerignore:3]: Re-included

**Review by Andrew-Folga — APPROVED** [review]
LGTM

**Review by alanplatt — APPROVED** [review]

**Pull request merged**